### ISSUE TITLE
Pick up and Move Core with Oct's

### ISSUE THREAD

**Issue by tshupp** [Anuken/Mindustry-Suggestions#1519]
I would like the ability to move the core using Oct's in sectors so I can place the core in a much more strategic location.

This will allow players to better collect resources in a sector without having to worry about there being limited access to the Core due to environmental obstructions. 

This is similar but extends upon the requests of #1432 and #1423 

**Before making this issue, place an `X` in the boxes below to confirm that you have acknowledged them.** *Failure to do so may result in your request being closed automatically.*

1. - [x] I have done a quick search in the list of suggestions to make sure this has not been suggested yet.
2. - [x] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
3. - [x] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
4. - [x] I have read `README.md` to make sure my idea is not listed under the "A few things you shouldn't suggest" category.

**Comment by GreyMario** [comment]
https://github.com/Anuken/Mindustry/blob/a544888f16f050890c280fb86103dcb447d99156/core/src/mindustry/world/blocks/storage/CoreBlock.java#L206

**Comment by Quick-korkc** [comment]
what do you think?
lets say the core is the brain
all block picked will disabled
if we can pick core so we can disable our own brain

**Comment by joshuaptfan** [comment]
1. Oct picks up core.
2. Oct gets shot down.
3. Instantly lose.

This suggestion is both OP and opens up a new griefing opportunity.

**Comment by itcannotbe** [comment]
But isn't static core location gives map makers control over the difficulty of the map.

And this suggestion takes that away.

Also OP AF.

**Comment by DeltaNedas** [comment]
uses an oct which good maps wont have available immediately
main issue is griefing by sending octs with core into enemy lines

**Comment by DarkBrick** [comment]
This would completly ruin ballancing and multiplayer, better not to do this

**Comment by TNT331** [comment]
Not an good idea AT ALL, too many malfunctions

**Comment by tshupp** [comment]
I will say yes this is OP AF for Multiplayer.  My main thought was within Campain where I have already captured the sector,  I would much rather move my core to the middle of the map to allow for optimal input/output as opposed to being on the edge where I have limited room to move items around. 

This would definitely require a lot of work concerning disabling the core.  Could probably add a new Attr of suspended when carried by an Oct. 

All this to say, I see your point and I agree that this is less than ideal for multiplayer scenarios.

Closing based on the overall consensus that this is OP as frick.
